src/stats.c

```diff
@@ -133,8 +133,16 @@ stats_arena_bins_print(void (*write_cb)(void *, const char *), void *cbopaque,
 			availregs = nregs * curslabs;
 			milli = (availregs != 0) ? (1000 * curregs) / availregs
 			    : 1000;
-			assert(milli <= 1000);
-			if (milli < 10) {
+
+			if (milli > 1000) {
+				/*
+				 * Race detected: the counters were read in
+				 * separate mallctl calls and concurrent
+				 * operations happened in between. In this case
+				 * no meaningful utilization can be computed.
+				 */
+				malloc_snprintf(util, sizeof(util), " race");
```

Comment:
We use %-5s for print so space should be fine. With numeric values we always output 5 chars. I tried a little bit locally and it looked a bit better with the space ("race" would be aligned with "util", otherwise it would be shifted by one char). Either case the alignment of other fields is not impacted. Any case the space would hurt?